nodejs/eks/cluster.ts

```diff
@@ -287,16 +288,16 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
     // Create the EKS service role
     let eksRole: pulumi.Output<aws.iam.Role>;
     if (args.serviceRole) {
-      eksRole = pulumi.output(args.serviceRole);
+        eksRole = pulumi.output(args.serviceRole);
     } else {
-      eksRole = (new ServiceRole(`${name}-eksRole`, {
-        service: "eks.amazonaws.com",
-        description: "Allows EKS to manage clusters on your behalf.",
-        managedPolicyArns: [
-            "arn:aws:iam::aws:policy/AmazonEKSClusterPolicy",
-            "arn:aws:iam::aws:policy/AmazonEKSServicePolicy",
-        ],
-      }, { parent: parent })).role;
+        eksRole = (new ServiceRole(`${name}-eksRole`, {
+            service: "eks.amazonaws.com",
+            description: "Allows EKS to manage clusters on your behalf.",
+            managedPolicyArns: [
+                "arn:aws:iam::aws:policy/AmazonEKSClusterPolicy",
+                "arn:aws:iam::aws:policy/AmazonEKSServicePolicy",
+            ],
+        }, { parent: parent })).role;
```

Comment:
Lots of whitespace changes from running `format` on this file - I tried undoing them a half dozen times to avoid dirtying this PR - but finally gave up - suggest reviewing with "Hide whitespace changes" turned on.